python/cudf/cudf/tests/test_categorical.py

```diff
@@ -553,6 +553,7 @@ def test_categorical_set_categories_categoricals(data, new_categories):
         pd.Series(["a", "1", "22", "1", "aa"]),
         pd.Series(["a", "1", "22", "1", "aa"], dtype="category"),
         pd.Series([1, 2, 3, 4], dtype="int64"),
+        pd.Series([1, 2, 3, 4], dtype="uint64"),
```

Comment:
Should some of these be negative? Doesn't seem like a valid use case for categorical, but maybe worth mentioning.